tokio-console/Cargo.toml

```
@@ -29,14 +29,27 @@ keywords = [
 [dependencies]
 atty = "0.2"
 console-api = { version = "0.6.0", path = "../console-api", features = ["transport"] }
-clap = { version = "3", features = ["cargo", "derive", "env"] }
+# Use 1.64.0 compatible version of `clap@4.1.14`.
+clap = { version = "=4.1.14", features = [
+    # these are all enabled by default
+    "std", "color", "help", "usage", "error-context", "suggestions",
+    # auto-wrap help text to fit the terminal width
+    "wrap_help",
+    # use Cargo env vars when determining help text
+    "cargo",
+    # enable derive attributes
+    "derive",
+    # parse values from env vars
+    "env"
+] }
+clap_complete = "=4.1.6"
```

This should also be 4.3 to match `clap` I believe.

```suggestion
clap_complete = "~4.3"
```